### DISCUSSION TITLE
Is it possible to set the first directory name of the current tabpage working directory as the tabpage name respectively?

### DISCUSSION THREAD

**Discussion by otakutyrant** [noib3/cokeline.nvim#120] [Q&A] [answered]
For example, `vi -p -c "tabdo lcd %:p:h" dotfiles nanozuki liuerfire` will show like:

![2023-06-25-014807_518x506_scrot](https://github.com/willothy/nvim-cokeline/assets/64188229/47eee1fb-3775-48a6-9b8e-5810bbd4644f)

After skimming the README, I was still not sure whether it is.

**Comment by willothy** [comment, marked as answer]
I'm not quite sure I understand the question, but nvim-cokeline does not currently support tabpages.
